src/NextGen/Gateways/Stripe/NextGenStripeGateway/NextGenStripeRepository.php

```diff
@@ -34,40 +36,52 @@ protected function generateStripePaymentIntent($accountId, Money $amount): Payme
      * Get or create Stripe Customer from Donation
      *
      * @unreleased
-     * @throws ApiErrorException
      * @throws Exception
+     * @throws ApiErrorException
      */
     public function getOrCreateStripeCustomerFromDonation(
         string $connectAccountId,
         Donation $donation
-    ): Customer {
+    ): Customer
+    {
         $donorCustomerId = give_stripe_get_customer_id($donation->email) ?? '';
```

Comment:
From what I can tell the `?? ''` is redundant. `give_stripe_get_customer_id()` uses an empty string as a default.

Reply:
Resolved 7f4ee705